pages/validators/requirements.md

```diff
@@ -17,4 +17,34 @@ For your node to successfully ingest bridge transactions from the relevant block
 💡IMPORTANT💡:The RPC endpoint you choose *MUST* satisfy the following requirements
 * supports `eth_chainId` method
 * supports `eth_getLogs` method
-* supports `"finalized"` as an input to `toBlock` parameter (Erigon is currently the only major Ethereum node software that does not support this.)
\ No newline at end of file
+* supports `"finalized"` as an input to `toBlock` parameter (Erigon is currently the only major Ethereum node software that does not support this.)
+
+# Required Configs
+
+💡IMPORTANT💡 The dYdX Chain has important node configurations required for normal chain operation. This includes:
+
+## `config.toml`
+
+```
+[consensus]
+timeout_commit = "500ms"
+```
+
+## `app.toml`
+
+```
+# Set up correct minimum-gas-price correctly. 
+# `12500000000adv4tnt` is for dYdX testnets. Recommended values may be different for production networks.
+minimum-gas-prices = "0.025ibc/8E27BA2D5493AF5636760E354E46004562C46AB7EC0CC4C1CA14E9E20E2545B5,12500000000adv4tnt"
```

Comment:
Using `12500000000` to be more consistent with recommendations [here](https://docs.dydx.trade/mainnet/genesis/starting_the_network#minimum-gas-price)